relayer/spy_relayer/src/listener/validation.ts

```diff
@@ -230,9 +28,22 @@ export type ParsedVaa<T> = {
 
 export type ParsedTransferPayload = {
   amount: BigInt;
-  originAddress: Uint8Array; //hex
+  originAddress: string; // hex
   originChain: ChainId;
-  targetAddress: Uint8Array; //hex
+  targetAddress: string; // hex
   targetChain: ChainId;
   fee?: BigInt;
 };
+
+/** Type guard function to ensure an object is of type ParsedTransferPayload */
+function IsParsedTransferPayload(
+  payload: any
+): payload is ParsedTransferPayload {
+  return (
+    typeof (payload as ParsedTransferPayload).amount == "bigint" &&
+    typeof (payload as ParsedTransferPayload).originAddress == "string" &&
+    typeof (payload as ParsedTransferPayload).originChain == "number" &&
+    typeof (payload as ParsedTransferPayload).targetAddress == "string" &&
+    typeof (payload as ParsedTransferPayload).targetChain == "number"
+  );
+}
```

Comment:
If this stays, I'll need to add a matching one for `IsParsedVaaWithParsedTransferPayload()` to do type guards on:

    ParsedVaa<ParsedVaaTransferPayload>

This might be necessary for the `Listener.validate()` return:
 https://github.com/certusone/wormhole/blob/3940c95bfb97f7866df92837d6b2f05b9591a24f/relayer/spy_relayer/src/backends/definitions.ts#L19-L20